### PR TITLE
Add docs for xml2json

### PR DESCRIPTION
**Pull request by hetunandu**

### CODE REVIEW

File: function-reference/xml2json.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description: Convert xml string to json string
+---
+
+# xml2json
+This uses the [xml-js](https://github.com/nashwaan/xml-js) library
+
+## Signature
+
+```text
+xml2json(xml: string, options: object) -> string
```
Comment (Nikhil-Nandagopal) [inline function-reference/xml2json.md:11]: @hetunandu the response should be a JSON object not a string right?

**Review by Nikhil-Nandagopal — APPROVED** [review]

File: function-reference/xml2json.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+description: Convert xml string to json string
```
Comment (Nikhil-Nandagopal) [inline function-reference/xml2json.md:2]: @hetunandu incorrect description

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
I have a single comment

File: function-reference/xml2json.md (hunk 2 above)
Comment (Nikhil-Nandagopal) [reply function-reference/xml2json.md]: @hetunandu this is still not resolved. We're not returning a json string but an object right?

File: function-reference/xml2json.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+# xml2json
+This uses the [xml-js](https://github.com/nashwaan/xml-js) library
```
Comment (Nikhil-Nandagopal) [inline function-reference/xml2json.md]: library also seems incorrect since we've updated the library

Comment (hetunandu) [reply function-reference/xml2json.md]: Looks like you reviewed an old version somehow. Check the latest changes

**Review by Nikhil-Nandagopal — APPROVED** [review]

File: function-reference/xmlParser.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+description: Convert xml string to json object
+---
+
+# xmlParser
+This uses the [xmlParser](https://github.com/NaturalIntelligence/fast-xml-parser) library
+
+## Signature
+
+```text
+xmlParser.parse(xml: string, options?: object) -> object
```
Comment (Nikhil-Nandagopal) [inline function-reference/xmlParser.md:11]: @hetunandu please add a usage example here

**Pull request merged**